api.swagger.yaml

```diff
@@ -9926,8 +9950,8 @@ components:
           description: 'This is topic of the feed, used for filtering'
           type: string
           enum:
-            - people
-            - groups
+            - user
```

Comment:
@sam-ludlow I think this change could cause mobile app issues, unless this is just a swagger change to reflect what is already in node?

Reply:
Yes node.js stays the same, only change here